test/config_errors.js

```diff
@@ -0,0 +1,12 @@
+import 'babel-core/register'
```

Comment:
This line is unnecessary - see https://github.com/carrot/roots-mini/pull/9/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R56